walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyARIbGzg1ObeVicxm6Txr1mqhUR4w12LQQ"></script>
```

Comment:
I think this commit is from a different PR?

Reply:
ah I think those were from the week 4 walkthroughs -- forgot to switch branches while making those changes.